### DISCUSSION TITLE
Zig Support - Go to definition and references

### DISCUSSION THREAD

**Discussion by idr4n** [zed-industries/zed#20689] [Feature Requests]
I have zig extension installed and it looks like some LSP features are working, but `go to definition` and `find all references` in particular is not working for me. Wondering whether this is and issue or I might be missing some configuration.

Thanks in advance!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
